armory/metrics/task.py

```diff
@@ -1587,10 +1587,10 @@ def _check_and_format(self, data):
     @staticmethod
     def relabel(unique_ids, list_of_id_arrays):
         """
-        Re-label int IDs to 1-indexed sequential IDs
+        Re-label int IDs to 0-indexed sequential IDs
         """
         unique_ids = sorted(set(unique_ids))
-        new_unique_ids = list(range(1, len(unique_ids) + 1))
+        new_unique_ids = list(range(len(unique_ids)))
         id_map = {k: v for k, v in zip(unique_ids, new_unique_ids)}
         new_list = [np.array([id_map[x] for x in array]) for array in list_of_id_arrays]
```

Comment:
is it important that the `new_list` ids are in sequential order? Or just that the range of ids is a consecutive span of ints? When I look at this variable, I notice it's not quite in order and not sure if your use of "sequential" in the comment implies it should be

```
(Pdb) new_list[2]
array([ 0, 21,  1,  3,  4,  5,  6,  7,  8, 24,  9, 10, 25, 11, 12, 13, 14,
       15, 22, 16, 17, 26, 18, 19, 20])
```

Reply:
Not for each array in the list. Just that the range of ids is consecutive ints (over the entire list of arrays). Reordering them in the arrays would require reordering them in the other data structures, which I would like to avoid.